boards/native: currently does not build for OSX

Native board currently doesn't build for OSX. Build process exits with

``` bash
ld: file too small (length=8) file 'RIOT/examples/hello-world/bin/native/drivers.a' for architecture i386
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make: *** [all] Error 1
```

With the patch below it compiles and seems to work as reported by @emmanuelsearch 

``` diff
diff --git a/cpu/native/Makefile.include b/cpu/native/Makefile.include
index ab8277f..211be07 100644
--- a/cpu/native/Makefile.include
+++ b/cpu/native/Makefile.include
@@ -1,2 +1,2 @@
export NATIVEINCLUDES += -I$(RIOTCPU)/native/include -I$(RIOTBASE)/sys/include
-export USEMODULE += periph drivers
+export USEMODULE += periph
```

@LudwigOrtmann any idea on this?

--- a/boards/native/Makefile.include
+++ b/boards/native/Makefile.include
@@ -16,7 +16,12 @@ export AR ?= $(PREFIX)ar
 export AS ?= $(PREFIX)as
 export LINK ?= $(PREFIX)gcc
 export SIZE ?= $(PREFIX)size
-export OBJCOPY = $(PREFIX)objcopy
+ifneq ($(shell uname -s),Darwin)
+export OBJCOPY ?= $(PREFIX)objcopy
+else
+export OBJCOPY ?= $(PREFIX)gobjcopy
+export OFLAGS ?= -O ihex
+endif
 
 export DEBUGGER = gdb
 export TERMPROG = $(ELF)

--- a/cpu/native/Makefile.include
+++ b/cpu/native/Makefile.include
@@ -1,2 +1,2 @@
 export NATIVEINCLUDES += -I$(RIOTCPU)/native/include -I$(RIOTBASE)/sys/include
-export USEMODULE += periph drivers
+export USEMODULE += periph

